src/RF24Logger.h

```diff
@@ -18,10 +18,25 @@
 #ifndef SRC_RF24LOGGER_H_
 #define SRC_RF24LOGGER_H_
 
-#include <WString.h>
+#include <String.h>

```

Comment:
We could do a 
```cpp
#if defined(ARDUINO_API_VERSION)
#include <String.h>
#else
#include <WString.h>
#endif
```

Reply:
You mean that sometimes must use ```WString.h``` and in other cases ```String.h```?
Interesting.
I have both files for Arduino NANO:
```
packages\arduino\hardware\avr\1.8.3\cores\arduino\Wstring.h
packages\arduino\tools\avr-gcc\7.3.0-atmel3.6.1-arduino7\avr\include\string.h
```
However the second file is ```string.h``` with lowercase. Does it matter when I use ```<String.h>``` or ```<string.h>```?

Let's try with your suggestion.

Reply:
On some non AVR platforms it still doesn't work:
```
  In file included from /home/runner/work/RF24Log/RF24Log/examples/AllLogLevelsLogger/AllLogLevelsLogger.ino:18:0:
  /home/runner/Arduino/libraries/RF24Log/src/RF24Logger.h:22:10: fatal error: String.h: No such file or directory
   #include <String.h>
            ^~~~~~~~~~
  compilation terminated.
```
Hm. I have built it locally against **arduino:samd:nano_33_iot** and it was working for me with **#include <String.h>**

Reply:
>Does it matter when I use <String.h> or <string.h>?

Hah, it matters. Now it looks nice when I included ```<string.h>``` (lowercase).